src/lemmyEvents/filterHandler.ts

```diff
@@ -19,8 +19,8 @@ export default class FilterHandler {
       console.log("Checking for match!");
       console.log(x);
       const found = checkText(x.words, [
-        postData.post.body || "",
-        postData.post.name,
+        postData.post.body.replace(/\*/g, '') || "",
+        postData.post.name.replace(/\*/g, ''),
```

Comment:
Why is it required that * should be replaced?

Reply:
and then why not on comments?

Reply:
I removed the * replace on that file